frontends/web/src/routes/account/summary/accountssummary.tsx

```diff
@@ -205,35 +205,45 @@ class AccountsSummary extends Component<Props, State> {
                     <Header title={<h2>{t('accountSummary.title')}</h2>}>
                         { debug && (
-                              exported ? (
-                                  <A href={exported} title={exported} className="flex flex-row flex-start flex-items-center">
-                                      <span>
-                                          <img src={checkIcon} style="margin-right: 5px !important;" />
-                                          <span>{t('account.openFile')}</span>
-                                      </span>
-                                  </A>
-                              ) : (
-                                  <a onClick={this.export} title={t('accountSummary.exportSummary')}>
-                                      <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="#699ec6" strokeWidth="2" strokeLinecap="round" strokeLinejoin="round">
-                                          <path d="M21 15v4a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2v-4"></path>
-                                          <polyline points="7 10 12 15 17 10"></polyline>
-                                          <line x1="12" y1="15" x2="12" y2="3"></line>
-                                      </svg>
-                                  </a>
-                              )
-                        )
-                        }
+                            exported ? (
+                                <A href={exported} title={exported} className="flex flex-row flex-start flex-items-center">
+                                    <span>
+                                        <img src={checkIcon} style="margin-right: 5px !important;" />
+                                        <span>{t('account.openFile')}</span>
+                                    </span>
+                                </A>
+                            ) : (
+                                <a onClick={this.export} title={t('accountSummary.exportSummary')}>
+                                    <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="#699ec6" strokeWidth="2" strokeLinecap="round" strokeLinejoin="round">
+                                        <path d="M21 15v4a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2v-4"></path>
+                                        <polyline points="7 10 12 15 17 10"></polyline>
+                                        <line x1="12" y1="15" x2="12" y2="3"></line>
+                                    </svg>
+                                </a>
+                            )
+                        )}
                     </Header>
                     <div className="innerContainer scrollableContainer">
                         <div className="content padded">
-                            {data && <Chart
-                                dataDaily={data.chartDataMissing ? undefined : data.chartDataDaily}
```

Comment:
why is this gone? the two fields are only valid (defined) if chartDataMissing is false, that's why it was like this. the backend might deliver garbage in this case

Reply:
fixed